cadl-tests/cadl/response.cadl

```diff
@@ -26,6 +27,9 @@ model Resource {
 
 model ResourceArray is Resource[];
 
+interface ResourceOperations
+  extends Azure.Core.ResourceOperations<NoRepeatableRequests & NoConditionalRequests & NoClientRequestId> {}
```

Comment:
This would be same as StandardResourceOperations, but that likely will be removed from core
https://github.com/Azure/cadl-azure/blob/main/packages/cadl-azure-core/lib/operations.cadl#L308